engine/seq/amulet.py

```diff
@@ -0,0 +1,48 @@
+import time
+
+from control import sos_ctrl
+from engine.seq.base import SeqBase
+from memory.combat_manager import combat_manager_handle
+
+combat_manager = combat_manager_handle()
+
+
+# TODO: Temporary code, moves along path, pausing while combat is active
+class SeqAmulet(SeqBase):
+    def __init__(
+        self,
+        name: str,
+    ):
+        super().__init__(name)
+        self.timer = 0.0
+        self.complete = False
+        self.steps = ["open_menu", "up_twice", "confirm", "confirm", "cancel", "exit"]
+        self.delay = 0.05
+
+    # Override
+    def execute(self, delta: float) -> bool:
+        step = self.steps.pop(0)
+        print(step)
+
+        match step:
```

Comment:
The match lacks a `default` case. In this particular case it will just default to `return False`, but it's kind of wobbly code as well.